auto_backup/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2004-2009 Tiny SPRL (<http://tiny.be>).
+# Copyright 2015 Agile Business Group <http://www.agilebg.com>
+# Copyright 2016 Grupo ESOC Ingenieria de Servicios, S.L.U. - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Database Auto-Backup",
+    "summary": "Backups database",
+    "version": "16.0.1.0.0",
```

Comment:
version should be "17.0.1.0.0"

Reply:
My bad
I forget look this
Thanks 
I fix it tomorrow